WalletWasabi/WabiSabi/Models/Serialization/JsonSerializationOptions.cs

```diff
@@ -34,7 +34,8 @@ public class JsonSerializationOptions
 				new IssuanceRequestJsonConverter(),
 				new CredentialPresentationJsonConverter(),
 				new ProofJsonConverter(),
-				new MacJsonConverter()
+				new MacJsonConverter(),
+				new ByteArrayJsonConverter()
```

Comment:
No. This contains the converters for WabiSabi protocol. Remove it please.
Wallet elements use decorations, no JsonSerializationOption